api/src/main/java/com/okta/sdk/client/AuthenticationScheme.java

```diff
@@ -26,14 +26,15 @@
  * The Authentication Scheme setting is helpful in cases where the code is run in a platform where the header information for
  * outgoing HTTP requests is modified and thus causing communication issues.
  * <p>
- * The SSWS (Okta session bearer token) should be used for the management SDK, {code NONE} should be used for non
- * authenticated requests.
+ * One of SSWS (Okta session bearer token) (or) OAUTH2 authentication schemes should be used for the management SDK, {code NONE}
+ * should be used for unauthenticated requests.
```

Comment:
This is a bit confusing to me.  Why are we using an authenttication scheme for unauthenticated requests?  

Also, what is `{code: NONE}`

Reply:
This should be `{@code NONE}` which should render to something like `<pre>NONE</pre>`.

That was likely my fault ;)

Reply:
Fixed